scenarios/aws/microVMs/microvms/resources/default/volume.xml

```diff
@@ -5,8 +5,8 @@
     <format type='qcow2'/>
     <permissions>
       <mode>0660</mode>
-      <owner>$(cat /etc/passwd | grep $USER | cut -d ':' -f 3)</owner>
-      <group>$(cat /etc/group | grep 'libvirt:' | cut -d ':' -f 3)</group>
+      <owner>$(grep $USER /etc/passwd | cut -d ':' -f 3)</owner>
+      <group>$(grep 'libvirt:' /etc/group | cut -d ':' -f 3)</group>
```

Comment:
```suggestion
      <owner>$(getent passwd $USER | cut -d ':' -f 3)</owner>
      <group>$(getent group libvirt | cut -d ':' -f 3)</group>
```

Reply:
This change was causing a malformed XML error. Left it as is for now.